pkgs/tools/nix/nixpkgs-manual-mmdoc/default.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenvNoCC, mmdoc, nixpkgs-manual-lib-docs }:
+
+stdenvNoCC.mkDerivation rec {
+  name = "nixpkgs-minimal-manual";
+
+  src = builtins.filterSource (path: type: type == "directory" || builtins.match ".*\.md" path == [] || builtins.match ".*\.dot" path == []) ../../../../doc;
+
+  phases = [ "buildPhase" ];
```

Comment:
Please don't overwrite phases in packages and instead disable the phases which break the build our are really not wanted.

Reply:
In this case probably only a `dontUnpack = true;` is needed

Reply:
```suggestion
  dontUnpack = true;
```